install/tools/ipa-ca-install

```diff
@@ -241,6 +243,14 @@ def install_master(safe_options, options):
         sys.exit("CA subject: {}".format(e))
 
     ca.install_check(True, None, options)
+
+    if not options.unattended:
+        ca.print_ca_configuration(options)
```

Comment:
Would you mind to print the CA configuration in both supervised and unattended mode?

Reply:
Sure, I'll push update tomorrow.  Thanks for reviewing!